tests/attacks/conftest.py

```diff
@@ -49,9 +49,9 @@ def get_image_classifier_list_for_attack(attack, defended=False, **kwargs):
             return None
 
         return [
-            potential_classier
-            for potential_classier in classifier_list
-            if attack.is_valid_classifier_type(potential_classier)
+            potential_classifier
```

Comment:
potential_classifier should probably be renamed as potential_estimator I guess?

Reply:
That's a good catch. I have left all testing functions with classifier instead of changing to estimator because for now we are still testing only with classifiers in the new system and I'm not sure yet for which functions we will need new functions for estimators here. We will update these names as soon as we introduce tests for attacks using estimators for testing.